File: College/7.c
/*

> Date Created: October 15, 2023
> Author: Ishaan Rastogi
> Purpose: WAP in C to check whether a given year is leap year or not
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working

*/

#include<stdio.h>
int main(void)
{  
    int year;
    printf("Enter year: ");
    scanf("%d", &year);

    if (year % 4 == 0)
        {
            printf("It's a leap year. \n");
        }
    else
    {
        printf("It's not a leap year. ");
    }
}

/* 

Ctrl + Shift + `

Terminal Code Loading...

gcc 7.c
./a.exe
 
*/

File: Web Development Projects/15 Intermediate JS/7.js
/*
> Date Created: June 10, 2024
> Author: Ishaan Rastogi
> Purpose: Operating entry & exit using arrays
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working
*/

var guestList = ["Anil Kapoor", "Tisca Chopra",, "Ashish Vidyarthi", "Neil Bhoopalam", "Sapna Pabbi", "Ishaan" ]; 

//Checks if your name is there or not
var guestName = prompt ("What is your name?");

if (guestList.includes(guestName)){
    alert("Welcome!");
} else {
    alert("Sorry, maybe next time.");
}

// Kicking Ishaan out of the guestList because he is not in the caste of 24.
guestList.pop; // Pops out the last index of array.

//But little did they know that Ishaan is the host for the interview. So, we have to add Ishaan back.
guestList.push("Ishaan");

/* 
For Terminal Code... Ctrl + Shift + `
*/